usecases/user/delete-conversation.md

```diff
@@ -4,75 +4,69 @@
             <strong>Name</strong>
         </td>
         <td>
-            lorem pisum
+            Delete conversation
         </td>
     </tr>
     <tr>
         <td>
             <strong>Actor</strong>
         </td>
         <td>
-            lorem pisum
+            User
         </td>
     </tr>
     <tr>
         <td>
             <strong>Description</strong>            
         </td>
         <td>
-            lorem pisum
+            The actor can delete a complete conversation.
         </td>
     </tr>
     <tr>
         <td>
             <strong>Pre-condition</strong>
         </td>
         <td>
-            lorem pisum
+            <ul>
+                <li>
+                    The actor must be logged in.
+                </li>
+                <li>
+                    There must be at least one conversation which can be deleted. 
+                </li>
+            </ul> 
         </td>
     </tr>
     <tr>
         <td>
             <strong>Scenario</strong>
         </td>
         <td>
-            <ul>
-                <li>
-                    Item 1
-                </li>
+            <ol>
                 <li>
-                    Item 2
+                    The actor deletes the conversation.
```

Comment:
How about selecting a conversation first and choosing an action, i.e. deletion, afterwards?